Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Contollers/NoticestoMarinersControllerTest.cs

```diff
@@ -16,22 +16,39 @@ public class NoticesToMarinersControllerTest
         private INMDataService _nMService;
         private ILogger<NoticesToMarinersController> _logger;
         private IHttpContextAccessor _contextAccessor;
+        private INMDataService _nMDataService;
+
```

Comment:
Please correct the folder name as Controllers from Contollers. Also update the namespace in [NoticestoMarinersControllerTest.cs]